File: src/pycropml/transpiler/antlr_py/tests/examples/SimplaceComponent/Simplace_SoilTemperature/src/java/Simplace_SoilTemperature/SoiltemperatureExogenous.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class SoilTemperatureExogenous
{
    private Double iAirTemperatureMax;
    private Double iTempMax;
    private Double iAirTemperatureMin;
    private Double iTempMin;
    private Double iGlobalSolarRadiation;
    private Double iRadiation;
    private Double iRAIN;
    private Double iCropResidues;
    private Double iPotentialSoilEvaporation;
    private Double iLeafAreaIndex;
    private Double [] SoilTempArray;
    private Double [] iSoilTempArray;
    private Double iSoilWaterContent;
    private Double iSoilSurfaceTemperature;
    
    public SoilTemperatureExogenous() { }
    
    public SoilTemperatureExogenous(SoilTemperatureExogenous toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.iAirTemperatureMax = toCopy.getiAirTemperatureMax();
            this.iTempMax = toCopy.getiTempMax();
            this.iAirTemperatureMin = toCopy.getiAirTemperatureMin();
            this.iTempMin = toCopy.getiTempMin();
            this.iGlobalSolarRadiation = toCopy.getiGlobalSolarRadiation();
            this.iRadiation = toCopy.getiRadiation();
            this.iRAIN = toCopy.getiRAIN();
            this.iCropResidues = toCopy.getiCropResidues();
            this.iPotentialSoilEvaporation = toCopy.getiPotentialSoilEvaporation();
            this.iLeafAreaIndex = toCopy.getiLeafAreaIndex();
            SoilTempArray = new Double[toCopy.getSoilTempArray().length];
        for (int i = 0; i < toCopy.getSoilTempArray().length; i++)
        {
            SoilTempArray[i] = toCopy.getSoilTempArray()[i];
        }
            iSoilTempArray = new Double[toCopy.getiSoilTempArray().length];
        for (int i = 0; i < toCopy.getiSoilTempArray().length; i++)
        {
            iSoilTempArray[i] = toCopy.getiSoilTempArray()[i];
        }
            this.iSoilWaterContent = toCopy.getiSoilWaterContent();
            this.iSoilSurfaceTemperature = toCopy.getiSoilSurfaceTemperature();
        }
    }
    public Double getiAirTemperatureMax()
    { return iAirTemperatureMax; }

    public void setiAirTemperatureMax(Double _iAirTemperatureMax)
    { this.iAirTemperatureMax= _iAirTemperatureMax; } 
    
    public Double getiTempMax()
    { return iTempMax; }

    public void setiTempMax(Double _iTempMax)
    { this.iTempMax= _iTempMax; } 
    
    public Double getiAirTemperatureMin()
    { return iAirTemperatureMin; }

    public void setiAirTemperatureMin(Double _iAirTemperatureMin)
    { this.iAirTemperatureMin= _iAirTemperatureMin; } 
    
    public Double getiTempMin()
    { return iTempMin; }

    public void setiTempMin(Double _iTempMin)
    { this.iTempMin= _iTempMin; } 
    
    public Double getiGlobalSolarRadiation()
    { return iGlobalSolarRadiation; }

    public void setiGlobalSolarRadiation(Double _iGlobalSolarRadiation)
    { this.iGlobalSolarRadiation= _iGlobalSolarRadiation; } 
    
    public Double getiRadiation()
    { return iRadiation; }

    public void setiRadiation(Double _iRadiation)
    { this.iRadiation= _iRadiation; } 
    
    public Double getiRAIN()
    { return iRAIN; }

    public void setiRAIN(Double _iRAIN)
    { this.iRAIN= _iRAIN; } 
    
    public Double getiCropResidues()
    { return iCropResidues; }

    public void setiCropResidues(Double _iCropResidues)
    { this.iCropResidues= _iCropResidues; } 
    
    public Double getiPotentialSoilEvaporation()
    { return iPotentialSoilEvaporation; }

    public void setiPotentialSoilEvaporation(Double _iPotentialSoilEvaporation)
    { this.iPotentialSoilEvaporation= _iPotentialSoilEvaporation; } 
    
    public Double getiLeafAreaIndex()
    { return iLeafAreaIndex; }

    public void setiLeafAreaIndex(Double _iLeafAreaIndex)
    { this.iLeafAreaIndex= _iLeafAreaIndex; } 
    
    public Double [] getSoilTempArray()
    { return SoilTempArray; }

    public void setSoilTempArray(Double [] _SoilTempArray)
    { this.SoilTempArray= _SoilTempArray; } 
    
    public Double [] getiSoilTempArray()
    { return iSoilTempArray; }

    public void setiSoilTempArray(Double [] _iSoilTempArray)
    { this.iSoilTempArray= _iSoilTempArray; } 
    
    public Double getiSoilWaterContent()
    { return iSoilWaterContent; }

    public void setiSoilWaterContent(Double _iSoilWaterContent)
    { this.iSoilWaterContent= _iSoilWaterContent; } 
    
    public Double getiSoilSurfaceTemperature()
    { return iSoilSurfaceTemperature; }

    public void setiSoilSurfaceTemperature(Double _iSoilSurfaceTemperature)
    { this.iSoilSurfaceTemperature= _iSoilSurfaceTemperature; } 
    
}

File: src/pycropml/transpiler/antlr_py/tests/examples/SimplaceComponent/Simplace_SoilTemperature/src/cs/Simplace_SoilTemperature/SoilTemperatureExogenous.cs
using System;
using System.Collections.Generic;

public class SoilTemperatureExogenous 
{
    private double _iAirTemperatureMax;
    private double _iTempMax;
    private double _iAirTemperatureMin;
    private double _iTempMin;
    private double _iGlobalSolarRadiation;
    private double _iRadiation;
    private double _iRAIN;
    private double _iCropResidues;
    private double _iPotentialSoilEvaporation;
    private double _iLeafAreaIndex;
    private double[] _SoilTempArray;
    private double[] _iSoilTempArray;
    private double _iSoilWaterContent;
    private double _iSoilSurfaceTemperature;
    
    public SoilTemperatureExogenous() { }
    
    
    public SoilTemperatureExogenous(SoilTemperatureExogenous toCopy, bool copyAll) // copy constructor 
    {
    if (copyAll)
    {
    
    _iAirTemperatureMax = toCopy._iAirTemperatureMax;
    _iTempMax = toCopy._iTempMax;
    _iAirTemperatureMin = toCopy._iAirTemperatureMin;
    _iTempMin = toCopy._iTempMin;
    _iGlobalSolarRadiation = toCopy._iGlobalSolarRadiation;
    _iRadiation = toCopy._iRadiation;
    _iRAIN = toCopy._iRAIN;
    _iCropResidues = toCopy._iCropResidues;
    _iPotentialSoilEvaporation = toCopy._iPotentialSoilEvaporation;
    _iLeafAreaIndex = toCopy._iLeafAreaIndex;
    SoilTempArray = new double[toCopy._SoilTempArray.Length];
            for (int i = 0; i < toCopy._SoilTempArray.Length; i++)
            { _SoilTempArray[i] = toCopy._SoilTempArray[i]; }
    
    iSoilTempArray = new double[toCopy._iSoilTempArray.Length];
            for (int i = 0; i < toCopy._iSoilTempArray.Length; i++)
            { _iSoilTempArray[i] = toCopy._iSoilTempArray[i]; }
    
    _iSoilWaterContent = toCopy._iSoilWaterContent;
    _iSoilSurfaceTemperature = toCopy._iSoilSurfaceTemperature;
    }
    }
    public double iAirTemperatureMax
        {
            get { return this._iAirTemperatureMax; }
            set { this._iAirTemperatureMax= value; } 
        }
    public double iTempMax
        {
            get { return this._iTempMax; }
            set { this._iTempMax= value; } 
        }
    public double iAirTemperatureMin
        {
            get { return this._iAirTemperatureMin; }
            set { this._iAirTemperatureMin= value; } 
        }
    public double iTempMin
        {
            get { return this._iTempMin; }
            set { this._iTempMin= value; } 
        }
    public double iGlobalSolarRadiation
        {
            get { return this._iGlobalSolarRadiation; }
            set { this._iGlobalSolarRadiation= value; } 
        }
    public double iRadiation
        {
            get { return this._iRadiation; }
            set { this._iRadiation= value; } 
        }
    public double iRAIN
        {
            get { return this._iRAIN; }
            set { this._iRAIN= value; } 
        }
    public double iCropResidues
        {
            get { return this._iCropResidues; }
            set { this._iCropResidues= value; } 
        }
    public double iPotentialSoilEvaporation
        {
            get { return this._iPotentialSoilEvaporation; }
            set { this._iPotentialSoilEvaporation= value; } 
        }
    public double iLeafAreaIndex
        {
            get { return this._iLeafAreaIndex; }
            set { this._iLeafAreaIndex= value; } 
        }
    public double[] SoilTempArray
        {
            get { return this._SoilTempArray; }
            set { this._SoilTempArray= value; } 
        }
    public double[] iSoilTempArray
        {
            get { return this._iSoilTempArray; }
            set { this._iSoilTempArray= value; } 
        }
    public double iSoilWaterContent
        {
            get { return this._iSoilWaterContent; }
            set { this._iSoilWaterContent= value; } 
        }
    public double iSoilSurfaceTemperature
        {
            get { return this._iSoilSurfaceTemperature; }
            set { this._iSoilSurfaceTemperature= value; } 
        }
}